lib/Models/Terria.js

```diff
@@ -363,6 +364,16 @@ Terria.prototype.start = function(options) {
         corsProxy.proxyDomains.push.apply(corsProxy.proxyDomains, config.proxyDomains);
         corsProxy.baseProxyUrl = that.configParameters.corsProxyBaseUrl;
 
+        var promises = [];
+
+        promises.push(loadJson(cp.proxyableDomainsUrl).then(function(proxyableDomains) {
```

Comment:
This means that if `proxyableDomainsUrl` is left at the default, you get this 404 every time you start up: ```GET http://localhost:3001/proxyabledomains/ 404 (Not Found)``` - are we OK with that?